packages/browser/src/index.bundle.replay.ts

```diff
@@ -10,5 +11,5 @@ Sentry.Integrations.Replay = Replay;
 Sentry.Integrations.BrowserTracing = BrowserTracing;
 
 export * from './index.bundle.base';
-export { BrowserTracing, addTracingExtensions, Replay, Feedback };
+export { BrowserTracing, addTracingExtensions, Replay, ReplayCanvas, Feedback };
```

Comment:
this means that we also include replay canvas in the .replay CDN bundles - do we want this?